main/accounts/index.ts

```diff
@@ -531,10 +531,17 @@ export class Accounts extends EventEmitter {
     }
   }
 
-  resolveRequest (req: AccountRequest) {
+  resolveRequest <T> (req: AccountRequest, result?: T) {
```

Comment:
what's the purpose of this typing here?  allowing a dynamic type of `result`?

Reply:
not really sure tbh, gonna remove it